package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@manifoldco/manifold-plan-table",
-  "version": "0.0.14",
+  "version": "0.0.1",
```

Comment:
Starting over at `0.0.1`! Based on feedback from James, though, I think we’ll cut a 1.0 soon. But I’d like to get a few more fixes in before then (after all, the name only changed today).